modules/network-observability-resource-recommendations.adoc

```diff
@@ -0,0 +1,20 @@
+//module included in the following assemblies:
+// network_observability/configuring_operator.adoc
+
+:_content-type: REFERENCE
+[id="network-observability-resource-recommendations_{context}"]
+= Resource management and performance recommendations
```

Comment:
The IBM SG dedicated a section to claims and recommendations. This page details the possible legal implications of stating recommendations in the docs.

See, [Claims and recommendations](https://www.ibm.com/docs/en/ibm-style?topic=information-claims-recommendations)

What about something like:

> Resource management and performance considerations

Reply:
@skrthomas  I'd leave it up to you. There are hundreds of instances of recommended/recommendation in the docs. 
 @dfitzmau  Good catch on the ISG!